app/models/item.rb

```diff
@@ -70,17 +70,20 @@ class Item < ApplicationRecord
     joins(:base_item)
       .where("lower(base_items.category) LIKE '%diaper%'")
       .where.not("lower(base_items.category) LIKE '%cloth%' OR lower(base_items.name) LIKE '%cloth%'")
+      .where.not("lower(base_items.category) LIKE '%adult%'")
   }
 
   scope :cloth_diapers, -> {
     joins(:base_item)
       .where("lower(base_items.category) LIKE '%cloth%' OR lower(base_items.name) LIKE '%cloth%'")
+      .where.not("lower(base_items.category) LIKE '%adult%'")
   }
 
   scope :adult_incontinence, -> {
     joins(:base_item)
       .where(items: { partner_key: %w(adult_incontinence underpads liners) })
-      .or(where("items.partner_key LIKE '%adult%' AND items.partner_key NOT LIKE '%cloth%'"))
+      .or(where("items.partner_key LIKE '%adult%' AND items.partner_key NOT LIKE '%cloth%' AND items.partner_key NOT LIKE '%wipes'"))
+      .or(where("lower(base_items.category) LIKE '%adult%' AND lower(base_items.category) NOT LIKE '%wipes%'"))
```

Comment:
This is starting to get really complex... can we document these scopes with examples? I am so ready for this and BaseItem to be gone so we can manage categories on the Item level.

Reply:
That's a reasonable request.    Or at least to say what we believe the business rules to be...    There is still some ambiguity and I don't know *why* we are sometimes using the partner_key.

Reply:
I'm also going to say that fixing all the scopes is out of scope of this PR.   *g*

Reply:
Have added some long-winded commentary, and sent off a couple of additional queries/suggestions to the business.

Reply:
I've got some more input from NDBN, so hold off on the final review for the moment.  Should get some changes in tomrorrow.